client/src/globalStyles.js

```diff
@@ -0,0 +1,17 @@
+import { createGlobalStyle } from "styled-components";
```

Comment:
You could improve your developer experience with https://styled-components.com/docs/tooling#babel-macro

```suggestion
import { createGlobalStyle } from "styled-components/macro";
```